jupyterhub/scopes.py

```diff
@@ -48,13 +47,13 @@
             'read:users:name',
             'read:users:groups',
             'read:users:activity',
+            'read:users:roles',
```

Comment:
Let's make read access to roles *not* a child of read:users. It's auth-related, so I think should be opt-in rather than part of a default role.